manifests/a/aluxnimm/outlookcaldavsynchronizer/4.3.0/aluxnimm.outlookcaldavsynchronizer.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: aluxnimm.outlookcaldavsynchronizer
+PackageVersion: 4.3.0
+PackageLocale: en-US
+Publisher: Generalize-IT Solutions OG
+PublisherUrl: https://caldavsynchronizer.org/
+PublisherSupportUrl: https://caldavsynchronizer.org/help/issues/
+PrivacyUrl: https://caldavsynchronizer.org/de/datenschutzerklaerung/
+# Author:
+PackageName: Outlook CalDav Synchronizer
```

```suggestion
PackageName: CalDavSynchronizer
```
This should match control panel